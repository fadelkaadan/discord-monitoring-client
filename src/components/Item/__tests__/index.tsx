import { render, screen, RenderResult, cleanup } from "@testing-library/react";
import Item from "../index";

describe("Item", () => {
  afterEach(cleanup);

  it("Should match the snapshot", () => {
    const { container }: RenderResult = render(
      <Item content="item" />
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  it("Should render a content", () => {
    const content = "item"
    render(<Item content={content} />);
    const text: HTMLElement = screen.getByText(content);
    expect(text.textContent).toEqual(content);
  });
});
