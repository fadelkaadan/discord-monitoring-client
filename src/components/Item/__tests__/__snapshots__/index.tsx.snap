// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item Should match the snapshot 1`] = `
<div
  class="sc-bdnxRM hdLxNf"
>
  <p>
    item
  </p>
  <button
    class="sc-gtsrHT bxDAAl"
  >
    x
  </button>
</div>
`;
